app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   has_many :posts
   validates :name, presence: true
   validates :posts_counter, numericality: { greater_than_or_equal_to: 0, only_integer: true }
-  def last_posts(id)
+  def last_posts
     Post.where(author: id).order(created_at: :desc).limit(3)
   end
```

Comment:
* Kindly work on these requirements, your app should function as the wireframe given below ⬇️ 
* Kindly make sure your app can handle these URLs below 
![image](https://user-images.githubusercontent.com/52497246/185039752-21c31f4d-c7eb-4222-9f86-ece0096e879b.png)

Reply:
Hi @Davidosky007. I checked the project and It's working. I think the problem is that you are testing without adding elements to the database. You should add some elements using the rails console like this: 

first_user = User.create!(name: 'Tom', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8', bio: 'Teacher from Mexico.', post_counter:0)
second_user = User.create!(name: 'Lilly', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8', bio: 'Teacher from Poland.', posts_counter:0)
first_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my first post', comments_counter: 0, likes_counter: 0)
second_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my second post', comments_counter: 0, likes_counter: 0)
third_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my third post', comments_counter: 0, likes_counter: 0)
fourth_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my fourth post', comments_counter: 0, likes_counter: 0)
fifth_post = Post.create!(author: first_user, title: 'Hello', text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis p', comments_counter: 0, likes_counter: 0)
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom first!' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom second!' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom third' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom fourth' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom fifth' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom sixth' )
Like.create!(post: first_post, author: second_user)

![Screenshot from 2022-08-17 07-58-45](https://user-images.githubusercontent.com/98363075/185140721-2160fe7e-cb7f-4540-a192-aab5a872d8d3.png)
![Screenshot from 2022-08-17 07-59-05](https://user-images.githubusercontent.com/98363075/185140725-23e71110-dc8f-4060-941b-c6ebb5cd8405.png)
![Screenshot from 2022-08-17 07-59-14](https://user-images.githubusercontent.com/98363075/185140728-cc9ef114-aef4-4046-ba62-74291f6cfff0.png)
![Screenshot from 2022-08-17 07-59-28](https://user-images.githubusercontent.com/98363075/185140730-7202045c-600b-4ed5-b25b-56187ac4823c.png)